### PR TITLE
1st test

### PR DESCRIPTION
**Pull request by pugnere**
Hi stijn,
thanks for the sandbox
PR discussions for include or not the proposition ;-)

### CODE REVIEW

File: test1.md [hunk 1]
```diff
@@ -0,0 +1 @@
+testing the workflow
```
Comment (stdweird) [inline test1.md]: @pugnere well let's make this a proper test 😄 
Please use capital at beginning of sentence and end with a `.`

You should be able to make the changes, make a new commit and push again.

**Comment by pugnere** [comment]
Well formed sentence.

**Comment by stdweird** [comment]
looks fine.

**Comment by stdweird** [comment]
@pugnere 

now in your local branch, do 
```bash
git checkout master
git pull upstream master
```
and you will see the merged commits from upstream master enter your master branch 

you should always do this when starting a new branch to work on; and you should always use a new branch when working on new issues.

**Pull request merged**